src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -315,6 +330,15 @@ public void setGoalSpeeds(ChassisSpeeds speeds, boolean fieldCentric) {
     }
   }
 
+  public void alignToFieldElement(Translation2d targetPosition) {
```

Comment:
Recommend using an enum here rather than Translation2d and then set a variable called something like `targetFieldElement` based on that enum